scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -987,6 +987,61 @@ def publish_visibility(self) -> PublishVisibility:
         return PublishVisibility(visibility) if visibility else None
 
 
+class GenericAssetConfig(Config):
+    """Generic Asset Config class."""
+
+    """
+    Example:
+
+    # Remote storage path for asset data
+    remote_path:
+        type: azureblob
+        storage_name: my_storage
+        container_name: my_container
+        container_path: foo/bar
+    # Local folder path to copy remote files into
+    local_path: ./prompt
+
+    """

```

Comment:
```suggestion
    """
    Args:
        file_name (Path): Prompt config file to load and validate.
```
Could also be added to ModelConfig as well